infra-core/src/main/java/com/redhat/service/smartevents/infra/core/models/gateways/Action.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.service.smartevents.infra.core.models.gateways;
```

Comment:
what about moving all the `gateways` to `v1`?

Reply:
Agreed. Done.